includes/managers/class-fs-admin-menu-manager.php

```diff
@@ -321,8 +321,7 @@ function is_submenu_item_visible( $id, $default = true, $ignore_menu_existence =
 			return fs_apply_filter(
 				$this->_module_unique_affix,
 				'is_submenu_visible',
-				$this->get_bool_option( $this->_default_submenu_items, $id, $default ),
-				$id
+				$this->get_bool_option( $this->_default_submenu_items, $id, $default )
```

Comment:
@bahiirwa please do see how to define a variable argument in PHPDoc (maybe a specific syntax for PHPStan)?

This for example, is a definite false positive. The `fs_apply_filter` expects at minimum 3 arguments, but can have any number of arguments greater or equal to 3. Please check the implementation.

Perhaps a specific syntax in the PHPDoc will make PHPStan understand this.

Also please do revert this false positive fix.